Core/Main/itkElastixRegistrationMethod.hxx

```diff
@@ -222,9 +223,15 @@ ElastixRegistrationMethod<TFixedImage, TMovingImage>::GenerateData()
 
     SetParameterValueAndWarnOnOverride(parameterMap, "ResultImagePixelType", fixedImagePixelTypeString);
 
-    // Initial transform parameter files are handled via arguments and enclosing loop, not
-    // InitialTransformParametersFileName
-    SetParameterValueAndWarnOnOverride(parameterMap, "InitialTransformParametersFileName", "NoInitialTransform");
+    // An initial transform parameter file may be specified by m_InitialTransformParameterFileName, and passed via the
+    // argument map (specifically the "-t0" argument), but not via the registration parameter map.
+    if (parameterMap.count("InitialTransformParametersFileName") != 0)
+    {
+      elx::log::warn(
+        "The parameter map entry for \"InitialTransformParametersFileName\" is ignored. "
+        "`ElastixRegistrationMethod` only supports specifying an initial transform parameter file by calling "
+        "`SetInitialTransformParameterFileName`");
+    }
```

Comment:
Discussed at the sprint. No need to produce a warning here, as agreed with Marius (@mstaring). The "InitialTransformParametersFileName" is not relevant for elastix registration parameter files, just ignore it. Konstantinos (@ntatsisk) asked to check if the "NoInitialTransform" wasn't there to avoid some warning/error log messages from elastix.